Assets/Scripts/Quest/QuestMonster.cs

```diff
@@ -46,8 +49,8 @@ public QuestMonster(QuestData.UniqueMonster qm) : base()
         }
 
         // Read info from quest data or base type
-        info = EventManager.SymbolReplace(qm.info);
```

Comment:
Ok. I will take this in consideration. Then the default symbol store should be the {name} notation at least for quests.
I will solve it in next localization commit.